### DISCUSSION TITLE
How can I import the ComponentPickerMenuPlugin?

### DISCUSSION THREAD

**Discussion by wywppkd** [facebook/lexical#5365] [Q&A]
Or maybe it only exists in Playground

<img width="837" alt="image" src="https://github.com/facebook/lexical/assets/20620574/386ff3cd-8d1a-49e6-8394-007a7fd611a5">

**Comment by 2wheeh** [comment]
I found It's implemented in Playground as custom plugin [here](https://github.com/facebook/lexical/blob/ee82c4f395a659eb36aa34ecd9e718894d5f094f/packages/lexical-playground/src/plugins/ComponentPickerPlugin/index.tsx#L316).
